client/payment-details/dispute-details/dispute-resolution-footer.tsx

```diff
@@ -273,6 +267,151 @@ const DisputeLostFooter: React.FC< {
 	);
 };
 
+const InquiryUnderReviewFooter: React.FC< {
+	dispute: Dispute;
+} > = ( { dispute } ) => {
+	const submissionDateFormatted = dispute.metadata.__evidence_submitted_at
+		? dateI18n(
+				'M j, Y',
+				moment
+					.unix(
+						parseInt( dispute.metadata.__evidence_submitted_at, 10 )
+					)
+					.toISOString()
+		  )
+		: '-';
+
+	return (
+		<CardFooter className="transaction-details-dispute-footer transaction-details-dispute-footer--primary">
+			<Flex justify="space-between">
+				<FlexItem>
+					{ createInterpolateElement(
+						sprintf(
+							/* Translators: %s - formatted date, <a> - link to documentation page */
+							__(
+								'You submitted evidence for this inquiry on %s. The cardholder’s bank is reviewing the case, which can take 120 days or more. You will be alerted when they make their final decision. <a>Learn more</a>.',
+								'woocommerce-payments'
+							),
+							submissionDateFormatted
+						),
+						{
+							a: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content -- Link content is provided by createInterpolateElement
+								<a
+									target="_blank"
+									rel="noopener noreferrer"
+									href="https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#inquiries"
+								/>
+							),
+						}
+					) }
+				</FlexItem>
+				<FlexItem className="transaction-details-dispute-footer__actions">
+					<Link
+						href={ getAdminUrl( {
+							page: 'wc-admin',
+							path: '/payments/disputes/challenge',
+							id: dispute?.id,
```

Comment:
A bit unnerving seeing that conditional chaining there – if there's no dispute ID we should bail a lot earlier (otherwise link will be broken). Not a big deal but I'd be surprised if this is needed.

Reply:
Agreed, this is redundant and is a copy-paste artefact.

Resolved in 04e1ea7